diff --git a/src/ckernels/wormhole/common/inc/ckernel_sfpu.h b/src/ckernels/wormhole/common/inc/ckernel_sfpu.h
index 7265a47e9..d9ae4466e 100644
--- a/src/ckernels/wormhole/common/inc/ckernel_sfpu.h
+++ b/src/ckernels/wormhole/common/inc/ckernel_sfpu.h
@@ -195,6 +195,7 @@ inline void sfpu_exp()
 template <int max_iter = 3, uint ADDR_MOD>
 inline void sfpu_reciprocal()
 {
+    for (;;);
     if constexpr (max_iter == 1)
     {
         // If we are only doing one iteration of the MAD loop, then we only need to use one LREG for the MAD instructions because we have our "first guess" in a hard-coded register
