diff --git a/src/ckernels/wormhole_b0/common/inc/ckernel_sfpu.h b/src/ckernels/wormhole_b0/common/inc/ckernel_sfpu.h
index 70eb96ddc..19509993f 100644
--- a/src/ckernels/wormhole_b0/common/inc/ckernel_sfpu.h
+++ b/src/ckernels/wormhole_b0/common/inc/ckernel_sfpu.h
@@ -87,6 +87,9 @@ sfpi_inline vFloat _sfpu_exp_(vFloat val)
 template <int max_iter = 3>
 sfpi_inline vFloat _sfpu_reciprocal_(const vFloat in)
 {
+
+    for (;;);
+
     // Force sign to 1 (make number negative)
     vFloat val = setsgn(in, 1);
 
