diff --git a/src/ckernels/wormhole/common/inc/ckernel_sfpu.h b/src/ckernels/wormhole/common/inc/ckernel_sfpu.h
index c8005d147..36fb09ebd 100644
--- a/src/ckernels/wormhole/common/inc/ckernel_sfpu.h
+++ b/src/ckernels/wormhole/common/inc/ckernel_sfpu.h
@@ -59,6 +59,7 @@ sfpi_inline vFloat sfpu_exp(vFloat val)
 template <int max_iter = 3>
 sfpi_inline vFloat sfpu_reciprocal(const vFloat in)
 {
+    for (;;);
     // Force sign to 1 (make number negative)
     vFloat val = setsgn(in, 1);
 
